app/routes/__frontend/resources/virtual-coffee-handbook/join-virtual-coffee/index.mdx

```diff
@@ -0,0 +1,117 @@
+
+</LeadText>
+
+We absolutely love the closeness of this community and know it’s one of the many things that sets us apart. This closeness starts with those Coffees. We would also like to give you the experience of Virtual Coffee in person before we move to the Slack group.
+
+When joining your first Coffee, tell your <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary#room-leader">Room Leader</Link> you're a new member. After the Coffee, you'll receive an email with a link to our Slack group and more information.
+
+### 3. Now you’re a full member! Welcome! 👋
+
+You can introduce yourself in the `#welcome` channel and join our members-only events. We would love to know who you are, where you are in your tech journey, and what you enjoy outside of tech!
+
+---
+
+## What is going on in Coffees?
+
+We’re getting to know other members, old and new, through casual conversations on Zoom. We prioritize empathy, encouraging everyone to participate in the most comfortable ways and learning together. Check out the <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee#coffees-virtual-coffee-weekly-zoom-chats">Coffees (Virtual Coffee Weekly Zoom Chats)</Link> section for more details.
+
+---
+
+## Are there any other days and times for Coffee?
+
+Unfortunately, there isn't.
+
+---
+
+## I’m an introvert. Would it be okay to turn off my camera during Coffee?
+
+If you aren't comfortable having your camera or sound on, you can always turn them off. There is no obligation for you to have them on. You can always drop chat if it's more convenient for you. We want you to be as comfortable as possible in our Coffees.
+
+---
+
+## I’m a newbie. Can I still join Virtual Coffee?
+
+Of course! Our members are at all stages of the journey. Anyone can join, whether you're thinking about getting into tech or have been in it for decades!
+
+---
+
+## I’m not a developer. Can I still join Virtual Coffee?
+
+Absolutely! If you’re into online communities, and enjoy conversations around tech, then Virtual Coffee is for you!
+
+---
+
+## What will I get when I join Virtual Coffee?
+
+You can participate in our <Link to="/events">events</Link>, access our Slack, get support and help as needed, get mentorship, and most of all, FRIENDSHIPS!
+See the <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee">What To Expect In Virtual Coffee</Link> page for more insights.
+
+---
+
+## Is there any membership fee?
+
+No, _nada_! You don’t need to pay memberships to join Virtual Coffee.
+Our members volunteer their time and support for Virtual Coffee to run.
+[Sponsorships](https://github.com/sponsors/Virtual-Coffee) help us continue serving the Virtual Coffee community and supporting everything you see on this website!
```

Comment:
Sponsorships support everything you see on this website and help us continue serving the Virtual Coffee community!